doc/GeoIP.conf.md

```diff
@@ -25,9 +25,10 @@ sensitive.
 
 * `AccountID` - Your MaxMind account ID. This was formerly known as
   `UserId`.
-* `DatabaseDirectory` - The directory to store the database files. If this
-  is not set, you must specify the directory using the `-d` command line
-  argument.
+* `DatabaseDirectory` - The directory to store the database files. If not
+  set, the default is `/usr/share/GeoIP` on Unix-like systems and
```

Comment:
:slightly_frowning_face:. It'd be nice if we could update this.

Speaking of this documentation, is there any standard way to install man files with Go releases?

Reply:
Do you mean update automatically? Yeah, unfortunate. Maybe we could have the ones in the repo have something like DATABASEDIR and replace them at package time. That means the website would not be so usable but that's close to what we have with the current geoipupdate and makes for a better installed experience.

I don't see goreleaser providing anything special for man pages. For tarballs it might just be best to include them as markdown. For building debs etc we can probably process to man at package time.

Reply:
Yeah, I mean substituting it out with the right value. I wonder if a make file or something is worth it for this and generating man pages for packagers, etc. This seems like it should be a solved problem.

Reply:
I added a Makefile. I included building the program in there too which I'm not sure makes sense. For goreleaser we wouldn't use that part at least, but we might be able to use the rest. Not really sure we can fully solve it ahead of time though.